lib/Service/RecipeService.php

```diff
@@ -1100,7 +1100,7 @@ public function parseRecipeFile($file) {
 	public function getRecipeImageFileByFolderId($id, $size = 'thumb'): File {
 		$recipe_folders = $this->root->getById($id);
 		if (count($recipe_folders) < 1) {
-			throw new Exception($this->il10n->t('Recipe with id %d not found.', [$id]));
+			throw new Exception($this->il10n->t('Recipe with ID %d not found.', [$id]));
```

Comment:
@Valdnet i have one question regarding this change: is it only syntactical to have a better English writing or is this one a technical thing (do we have a duplicate strong here somewhere)

Reply:
If the ID is a direct identifier and points to an individual, the item or item should be in uppercase.